docs/websub.md

```diff
@@ -8,6 +8,14 @@ In MOSIP WebSub is used to share data with services and partners.  [Kafka](https
 
 ![](_images/websub.png)
 
+The relationship of Websub with other services is explained here.NOTE: The numbers do not signify sequence of operations or control flow
+
+1. Topic is registered and published or subscribed by Mosip Services.
+2. Content delivery, Intent Verification is done Websub to Mosip Services.
```

Comment:
Sentence needs correction - Content delivery, Intent Verification is done Websub to Mosip Services.

Reply:
done